code/modules/client/preference_setup/loadout/lists/utility.dm

```diff
@@ -84,9 +85,8 @@ modular computers
 	path = /obj/item/modular_computer/tablet
 	cost = 4
 
-/datum/gear/utility/customtablet/New()
-	..()
-	gear_tweaks += new /datum/gear_tweak/tablet()
+/datum/gear/utility/customtablet/get_gear_tweak_options()
+	. = ..() | /datum/gear_tweak/tablet
```

Comment:
Perhaps Lazyinit .[/datum/gear_tweak/tablet] for consistency

Reply:
It needs to be non-associative if I want it to be 1:1 with previous implementation.